### ISSUE TITLE
Suggestion - Pin the false positive antivirus alerts

### ISSUE THREAD

**Issue by rinze24** [svenstaro/miniserve#1414]
When I download the exe file from release, firefox, edge and other browser alert me for potential virus.
**Found #1210 that this is false positive.**

Suggestion for clarification:

- Like in UPX issue tracker they pinned it for easy viewing.
- Release unpacked of the tool.

Thanks for this wonderful tool, just got a bit startled when virustotal report 35 hits.
`miniserve-0.27.1-x86_64-pc-windows-msvc.exe`
https://www.virustotal.com/gui/file/060705db94b7ea673486db72ffc54fbd5e3c898758f9a16575d636c78e95dffb

**Comment by svenstaro** [comment]
It might be better to put a note in the README next to the download link to the Windows binary?

**Comment by rinze24** [comment]
Yes, anything will do as long as there is some clarification with the situation.

**Comment by svenstaro** [comment]
Would you like to add that note to the README?
